### PR TITLE
UI: responsive hero banner

### PR DESCRIPTION
**Pull request by tetianaman**
I keep banner height full-screen  for all screens:
`--bannerHeight: calc(100vh - var(--headerHeight) - var(--marginBottom));`

### CODE REVIEW

File: src/styles/common.scss [hunk 1]
```diff
@@ -142,6 +142,66 @@
...
+        margin-left: 2vw;
+        margin-right: 2vw;
+    }
+}
+
+@media (max-width: $tablet-width) {
+    .heroBanner {
+        --marginBottom: 8vw;
+        margin-left: 6vw;
+        margin-right: 6vw;
```
Comment (Letushev) [inline src/styles/common.scss:201]: I don't understand why the hero banner gets thinner as the viewport becomes smaller. Maybe let's not add side margins at all?

**Review by Letushev — COMMENTED** [review]
I see in the original UI it takes around 50vh from $tablet-width to $desktop-width

Screenshot 2024-09-23 at 18 36 30

**Review by Letushev — APPROVED** [review]

File: src/styles/common.scss (hunk 1 above)
Comment (tetianaman) [reply src/styles/common.scss:201]: I've put margins to all home page section by wix template design so far.

**Comment by tetianaman** [comment]
> I see in the original UI it takes around 50vh from $tablet-width to $desktop-width

I decided to fit it to screen, because I don't know what height to take. I think it should depends on some tablet dimensions to look good. To keep it stretched much easier and looks good.

**Pull request merged**